rcl_yaml_param_parser/src/parser.c

```diff
@@ -169,8 +169,7 @@ static rcl_ret_t add_name_to_ns(
       sep_str = PARAMETER_NS_SEPERATOR;
       break;
     default:
-      res = RCL_RET_ERROR;
-      break;
+      return RCL_RET_ERROR;
```

Comment:
Could you also remove the following `if (RCL_RET_OK == res)` condition and the `res` variable all together? I'd just do it myself, but I wanted to see what you think about it.